src/widget/wlibrarysidebar.cpp

```diff
@@ -205,6 +205,22 @@ void WLibrarySidebar::keyPressEvent(QKeyEvent* event) {
     //    // encoder click via "GoToItem"
     //    qDebug() << "GoToItem";
     //    TODO(xxx) decide what todo here instead of in librarycontrol
+    } else if (event->key() == Qt::Key_Left) {
+        auto selModel = selectionModel();
+        QModelIndexList selectedRows = selModel->selectedRows();
+        // If an expanded item is selected let QTreeView collapse it
+        if (isExpanded(selectedRows.at(0))) {
```

Comment:
Accessing `selectedRows.at(0)` before checking that it is not empty is wrong.

I guess we need to `return` early if `selectedRows.isEmpty()`. Otherwise continue and decide what to do depending on `firstIndex = selectedRows.first()`.

Reply:
right, I'll add the empty check.
For some reason I remembered `isEpanded` would do the `isValid` check itself, which is wrong.